tests/FakeItEasy.Specs/FakeItEasy.Specs.csproj

```diff
@@ -24,7 +24,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>TRACE;DEBUG;FEATURE_SELF_INITIALIZED_FAKES</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;FEATURE_SELF_INITIALIZED_FAKES FEATURE_EVENT_ARGS_MUST_EXTEND_EVENTARGS</DefineConstants>

```

Comment:
I noticed, but when I looked at the help, it told me to separate by spaces, but the IDE added the semicolons after TRACE and DEBUG. I now see I was looking at the VS2012 help (why this is the default page when I hit F1 in VS2015, I could not tell you). 2015's help does say to use semicolons. Will fix.